src/cljs/org/broadinstitute/firecloud_ui/common.cljs

```diff
@@ -136,18 +136,24 @@
    :queued (apply + (map workflowCountsByStatus ["Queued" "Launching"]))
    :active (apply + (map workflowCountsByStatus ["Submitted" "Running" "Aborting"]))})
 
-(def root-entity-types
-  ["participant" "sample" "pair" "participant_set" "sample_set" "pair_set"])
+(def root-entity-types ["participant" "sample" "pair" "participant_set" "sample_set" "pair_set"])
+(def singular->set {"participant" "participant_set"
```

Comment:
I like the name here but it is inconsistent with `set-type->attribute`.  Could you rename that to `set->singular` or make consistent some other way?